src/Spout/Writer/Common/Entity/Row.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Box\Spout\Writer\Common\Entity;
+
+use Box\Spout\Writer\Common\Entity\Style\Style;
+use Box\Spout\Writer\Common\Manager\Style\StyleMerger;
+
+class Row
+{
+    /**
+     * The cells in this row
+     * @var array
+     */
+    protected $cells = [];
+
+    /**
+     * The row style
+     * @var null|Style
+     */
+    protected $style = null;
+
+    /**
+     * @var StyleMerger
+     */
+    protected $styleMerger;
+
+    /**
+     * Row constructor.
+     * @param array $cells
+     * @param Style|null $style
+     */
+    public function __construct(array $cells = [], Style $style = null)
```

Comment:
Also, I would make the `$cells` param a required param. If you want an empty row, then just pass an empty array yourself.

Regarding the 2nd argument, should we just allow setting the row style using `Row::setStyle()`? Or is it better to match the `Cell` signature?